host_cleanup.sh

```diff
@@ -6,13 +6,13 @@ source lib/common.sh
 
 # Kill and remove the running ironic containers
 for name in ironic ironic-inspector dnsmasq httpd mariadb; do
-    sudo podman ps | grep -w "$name$" && sudo podman kill $name
-    sudo podman ps --all | grep -w "$name$" && sudo podman rm $name -f
+    sudo ${CONTAINER_RUNTIME} ps | grep -w "$name$" && sudo ${CONTAINER_RUNTIME} kill $name
```

Comment:
quotes around variables.
same for other places.

Reply:
leaving it as is for now, we will fix the style in the whole file afterwards